CHANGES/5771.feature

```diff
@@ -0,0 +1 @@
+Allow users to sync content in mirror mode
```

Comment:
s/allow/enable

Reply:
@lubosmj please add this to the Feature list also

Reply:
I believe It is already mentioned in the feature list, see the first bullet point https://github.com/pulp/pulp_container/blob/master/docs/index.rst#features

Reply:
it is a bit confusing, mirror in that context means - sync, you can mirror in additive mode and mirror mode. We can re-phrase - Sync container image repositories hosted on Docker-hub,...in mirror or additive mode,